src/Universum/Functor/Compose.hs

```diff
@@ -0,0 +1,21 @@
+{-# LANGUAGE Safe #-}
+
+module Universum.Functor.Compose
```

Comment:
I thought a little bit more about this module. And I think it's better to name it `Universum.Functor.Fmap`. Content of this module now more related to `fmap` rather than composition...